app/presenters/document/web_resource_presenter.rb

```diff
@@ -154,11 +154,10 @@ def playable?
 
     def downloadable?
       if url.blank? ||
-          mime_type.blank? ||
-          download_disabled? ||
-          media_type == 'iiif' ||
-          (media_type == 'text' && mime_type == 'text/plain; charset=utf-8') ||
-          (media_type == 'video' && mime_type == 'text/plain; charset=utf-8')
+        download_disabled? ||
+        media_type == 'iiif' ||
+        (media_type == 'text' && mime_type == 'text/plain; charset=utf-8') ||
```

Comment:
Use 4 (not 2) spaces for indenting a condition in an `if` statement spanning multiple lines.